pkg/detectors/ayrshare/ayrshare.go

```diff
@@ -48,7 +49,7 @@ func (s Scanner) FromData(ctx context.Context, verify bool, data []byte) (result
 		}
 
 		if verify {
-			req, err := http.NewRequestWithContext(ctx, "GET", "https://app.ayrshare.com/api/analytics/links", nil)
+			req, err := http.NewRequestWithContext(ctx, "GET", "https://app.ayrshare.com/api/user", nil)
```

Comment:
The `/user` API returns useful information, such as the user's email, which can be included in the extra data. 
I also recommend reviewing the [status code documentation](https://docs.ayrshare.com/rest-api/errors/http-status-codes) to handle responses appropriately.

Reply:
done